astroquery/cadc/core.py

```diff
@@ -38,7 +38,7 @@
 
 CADC_COOKIE_PREFIX = 'CADC_SSO'
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('cadc')
```

Comment:
While I find this nice, it's very inconsistent with the rest of the modules that don't set the name of the module. I think all should, but more importantly, be consistent for all the modules (see sibling comment under gemini)

Reply:
This is the best I can do here. I can look again at the issue I had with having the `cadc` logger use the `astroquery` one.